unit_tests/tests/test_block_number.rs

```diff
@@ -0,0 +1,18 @@
+mod common;
+use common::*;
+
+use std::collections::HashMap;
+
+use starknet::providers::{jsonrpc::{HttpTransport, JsonRpcClient}, Provider};
```

Comment:
same

Reply:
Renamed all 'alchemy' occurrences to 'pathfinder' in commit 185e58ed0bc13217e543784408673836807e87d2. This include `secret.json`, so make sure to update you key names in there!